eth_account/_utils/validation.py

```diff
@@ -48,6 +50,24 @@ def is_empty_or_checksum_address(val):
         return is_valid_address(val)
 
 
+def is_access_list(val):
+    """Returns true if 'val' is a valid access list."""
+    if not is_list_like(val):
+        return False
+    for item in val:
+        if not is_list_like(item):
+            return False
+        if len(item) != 2:
+            return False
+        address, storage_keys = item
+        if not is_address(address):
+            return False
+        for storage_key in storage_keys:
+            if not is_int_or_prefixed_hexstr(storage_key):
+                return False
+    return True
+
+
 TRANSACTION_FORMATTERS = {
```

Comment:
@malonaz Adding to the clarification comment I think `TRANSACTION_FORMATTERS` and `TRANSACTION_VALID_VALUES` here could use renaming to reflect that this is for legacy transaction values rather than generic naming since we now have differing txn types.